tests/mmm/test_transformers.py

```diff
@@ -142,6 +143,15 @@ def test_geometric_adstock_x_zero(self, mode):
             (np.linspace(start=0.0, stop=1.0, num=50), 0.8, 50),
         ],
     )
+    def test_binomial_adstock(self, x, alpha, l_max):
+        y = binomial_adstock(x=x, alpha=alpha, l_max=l_max)
+        y_np = y.eval()
+        w1 = (1 - 1 / (l_max + 1)) ** (1 / alpha - 1)
+        w2 = (1 - 2 / (l_max + 1)) ** (1 / alpha - 1)
+        assert y_np[0] == x[0]
+        assert np.testing.assert_almost_equal(y_np[1], x[1] + w1 * x[0])
+        assert np.testing.assert_almost_equal(y_np[2], x[2] + w1 * x[1] + w2 * x[0])
```

Comment:
I think you just need ` np.testing.assert_almost_equal`, you can remove the `asserts`